docs/docs/getting-started-google-compute-enging/index.md

```diff
@@ -0,0 +1,137 @@
+
+Sign up for Google Cloud Platform by going to [https://cloud.google.com/](https://cloud.google.com/). Once there, click 'Try It Free'. This will take you to a sign up wizard. Go through it (select your country, agree to terms, etc). At this point you'll be prompted for your personal information and a credit card. Once you offer all that, you'll be signed up and signed in!
+
+> Once you are signed up, you will be prompted to Try Compute Engine, and will be offered a 15 minute quickstart. It might be a good idea to go through this process to get a solid understanding of the platform.
+
+***
+
+## Start a Project <a name="start-project"></a>
+
+Google Cloud Platform starts you out in a default 'project' called My First Project. Projects are just a way of organizing your resources to be associated with the project they are used for. 
+
+You might not want to stick with this default project, so go ahead and create a new one:
+
+1. From the Dashboard or Home screen, you'll see a drop-down at the top of the screen with the name of your project displayed (defaulted to My First Project). Click that to open the menu.
+2. Click Create Project.
+3. This will open a dialog screen for you to name your project. Click Show Advanced Options to select your Region. This is important even if you don't change the region, because you'll be asked for the Region later when setting up credentials for your Stack.
+
+> Take note not only of the Project Name but the Project ID. You will need this ID when creating credentials for your Koding Stack. 
+
+***
+
+## Create your Service Account Key <a name="create-key"></a>
+
+Your Google Cloud Platform Service Account Key is a JSON file that you'll need to save to be used later as credentials for Koding to utilize GCE as your provider. 
+
+To generate your key:
+
+1. [Login](https://cloud.digitalocean.com/login) to Google Cloud Platform.
+2. Go to the Credentials screen inside the API Manager. [Here is a direct link](https://console.cloud.google.com/apis/credentials)
+3. On the Credentials tab (this is the default tab), click Create Credentials drop-down.
+4. Choose Service Account Key from the list.
+5. Set the Service Account to 'Compute Engine default service account'.
+6. Set the Key Type to JSON.
+7. Click Create.
+
+This will save your Key as a file to your computer. 
+
+> Make sure this is successful (that the file is saved successfully and that you can find the file). You'll need this later when you create your first Stack. 
+
+
+***
+
+## Create your Stack <a name="create-stack"></a>
+
+To create your Stack:
+
+1. [Login to Koding](https://koding.com/Teams/Select)
+2. Go to your [Dashboard](https://relepic.koding.com/Home) OR click [Stacks](https://relepic.koding.com/Home/Stacks) from your sidebar.
```

Comment:
These links are to a specific team.  You can't link to a person's dashboard directly from the docs.  Make sure this is also the case with other dashboard links.